docs/core/development/policy/code-dev.rst

```diff
@@ -1,277 +1,6 @@
-
-
-
-
-This depends upon the ``.py`` file having an appropriate "test-case-name" tag that indicates which test cases provide coverage.
-See the :doc:`Test Standards <test-standard>` document for
-details about using "test-case-name" . In this example, the ``twisted.mail.test.test_imap`` test will be run.
-
-
-
-
-Many tests create temporary files in /tmp or ./_trial_temp, but
-everything in /tmp should be deleted when the test finishes. Sometimes these
-cleanup calls are commented out by mistake, so if you see a stray ``/tmp/@12345.1`` directory, it is probably from ``test_dirdbm`` or ``test_popsicle`` .
-Look for an ``rmtree`` that has been commented out and complain to
-the last developer who touched that file.
-
-
-
-
-
-Building docs
--------------
-
-Twisted documentation (not including the automatically-generated API docs) is generated by `Sphinx <https://sphinx-doc.org/>`_ .
-The docs are written in Restructured Text (``.rst``) and translated into ``.html`` files by the ``bin/admin/build-docs`` script.
-
-To build the HTML form of the docs into the ``doc/`` directory, do the following:
-
-.. code-block:: console
-
-    ./bin/admin/build-docs .
-
-
-Committing and Post-commit Hooks
---------------------------------
-
-
-Twisted's Trac installation is notified when the Git repository changes,
-and will update the ticket depending on the Git commit logs.
-When making a branch for a ticket, the branch name should end
-in ``-<ticket number>`` , for
-example ``my-branch-9999`` . This will add a ticket comment containing a
-changeset link and branch name. To make your commit message show up as a comment
-on a Trac ticket, add a ``refs #<ticket number>`` line at the
-bottom of your commit message. To automatically close a ticket on Trac
-as ``Fixed`` and add a comment with the closing commit message, add
-a ``Fixes: #<ticket number>`` line to your commit message. In
-general, a commit message closing a ticket looks like this:
-
-
-
-
-
-::
-
-    
-    Merge my-branch-9999: A single-line summary.
-    
-    Author: jesstess
-    Reviewers: exarkun, glyph
-    Fixes: #9999
-    
-    My longer description of the changes made.
-
-
-
-
-The :doc:`Twisted Coding Standard <coding-standard>` 
-elaborates on commit messages and source control.
-
-
-
-
-
-Emacs
------
-
-
-
-A minor mode for development with Twisted using Emacs is available.  See ``twisted-dev.el`` , provided by `twisted-emacs <https://launchpad.net/twisted-emacs>`_ ,
-for several utility functions which make it easier to grep for methods, run test cases, etc.
-
-
-
-
-
-Building Debian packages
-------------------------
-
-
-
-Our support for building Debian packages has fallen into disrepair.  We
-would very much like to restore this functionality, but until we do so, if
-you are interested in this, you are on your own.  See `stdeb <https://github.com/astraw/stdeb>`_ for one possible approach to
-this.
+The content of this page was moved to :doc:`</core/development/dev-process>`.
```

Comment:
👍 Thanks for leaving the redirect in place, I was just looking to comment about this :)